<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMaterialDialog</class>
 <widget class="QDialog" name="NewMaterialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Material name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newMaterialName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Young modulus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newModulusButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse Young Modulus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select K (Bulk modulus)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newKButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse K</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Poisson's ratio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newMaterialPoisson">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="newMaterialDensity">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Density [t/mm3]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addMaterialButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Material</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newGButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Brows G</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select G (Shear modulus)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select tau (specific time)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newTauButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Brows tau</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
